revenue_app/urls.py

```diff
@@ -8,14 +8,15 @@
     OrganizerTransactions,
     OrganizersTransactions,
     TransactionsByDate,
+    TransactionsSearch,
 )
 
 
 urlpatterns = [
     url(r'^$', Dashboard.as_view(), name='dashboard'),
     url(r'transactions/$', OrganizersTransactions.as_view(), name='organizers-transactions'),
     url(r'organizers/(?P<organizer_id>[0-9]+)/$', OrganizerTransactions.as_view(), name='organizer-transactions'),
-    # url(r'organizers/search/$', OrganizerTransactions.as_view(), name='organizer-transactions'),
+    url(r'transactions/search/$', TransactionsSearch.as_view(), name='transactions-search'),
```

Comment:
question: is this view intended to receive other params in the future?

Reply:
Yes, maybe it could receive the event id or a date. Now we are just using it for searching by email.